server/src/main/java/org/apache/gravitino/server/web/rest/PermissionOperations.java

```diff
@@ -40,8 +40,10 @@
 import org.apache.gravitino.lock.LockType;
 import org.apache.gravitino.lock.TreeLockUtils;
 import org.apache.gravitino.metrics.MetricNames;
+import org.apache.gravitino.server.authorization.NameBindings;
 import org.apache.gravitino.server.web.Utils;
 
+@NameBindings.AccessControlInterfaces
```

Comment:
Please add comments to describe the Class `NameBindings`.
I don't know its purpose.

Reply:
Added.